### PR TITLE
Update services.json

### PR DESCRIPTION
**Pull request by rootmelo92118**
TLD onion is only used in tor. And tor DNS won't be regulated by AdGuard Home. It doesn't make sense to add this kind of domain to this file. By the way, the domain one.one is not belonging to Cloudflare, but the one.one.one is.

https://bgp.he.net/dns/one.one
https://bgp.he.net/dns/one.one.one

source : 
https://github.com/v2fly/domain-list-community/blob/master/data/ebay
https://github.com/v2fly/domain-list-community/blob/master/data/discord
https://github.com/v2fly/domain-list-community/blob/master/data/youtube
https://github.com/v2fly/domain-list-community/blob/master/data/bilibili
https://github.com/v2fly/domain-list-community/blob/master/data/telegram
https://github.com/v2fly/domain-list-community/blob/master/data/cloudflare
https://github.com/v2fly/domain-list-community/blob/master/data/douban
https://github.com/v2fly/domain-list-community/blob/master/data/epicgames
https://github.com/v2fly/domain-list-community/blob/master/data/facebook
https://github.com/v2fly/domain-list-community/blob/master/data/facebook-dev
https://github.com/v2fly/domain-list-community/blob/master/data/messenger
https://github.com/v2fly/domain-list-community/blob/master/data/amazon
https://github.com/v2fly/domain-list-community/blob/master/data/amazontrust
https://github.com/v2fly/domain-list-community/blob/master/data/amazon-ads
https://github.com/v2fly/domain-list-community/blob/master/data/aws
https://github.com/v2fly/domain-list-community/blob/master/data/imdb
https://github.com/v2fly/domain-list-community/blob/master/data/kindle
https://github.com/v2fly/domain-list-community/blob/master/data/primevideo

### CODE REVIEW

File: assets/services.json [hunk 1]
```diff
@@ -13,81 +13,297 @@
...
+        "||containersonaws.com^",
         "||createspace.com^",
-        "||aws"
+        "||elasticbeanstalk.com^",
+        "||gameon-masters.com^",
+        "||gdansk-amazon.com^",
+        "||images-amazon.com^",
+        "||imdb.com^",
+        "||imdb.to^",
+        "||imdb^",
```
Comment (ameshkov) [inline assets/services.json:157]: `imdb` TLD?

File: assets/services.json [hunk 2]
```diff
@@ -13,81 +13,297 @@
...
+        "||ssl-images-amazon.com^",
+        "||thinkboxsoftware.com^",
+        "||ueberamazon.de^",
+        "||xn--cckwcxetd^",
+        "||xn--jlq480n2rg^",
+        "||yamaxun.cn^",
+        "||yamaxun.com^",
+        "||yamaxun^",
+        "||z.cn^",
+        "||zappos^",
```
Comment (ameshkov) [inline assets/services.json:207]: Zappos TLD?

File: assets/services.json [hunk 3]
```diff
@@ -13,81 +13,297 @@
...
+        "/.+\\.awsdns-[0-9][0-9]\\.co\\.uk$/",
```
Comment (ameshkov) [inline assets/services.json:208]: Can this be replaced with `||awsdns-*.co.uk^`?

File: assets/services.json [hunk 4]
```diff
@@ -13,81 +13,297 @@
...
+        "/.+\\.awsdns-[0-9][0-9]\\.com$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.net$/",
+        "/.+\\.awsdns-[0-9][0-9]\\.org$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.biz$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.com$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.net$/",
+        "/.+\\.awsdns-cn-[0-9][0-9]\\.top$/",
+        "/.+\\.awsdns-cn-[0-9][a-e0-9]\\.cn$/",
+        "/^images-cn-8\\.ssl-images-amazon\\.com$/",
```
Comment (ameshkov) [inline assets/services.json:217]: Replace with a regular rule: `||images-cn-8.ssl-images-amazon.com^`

Please avoid regex rules when possible.

File: assets/services.json [hunk 5]
```diff
@@ -13,81 +13,297 @@
...
+        "||bilivideo.net^",
         "||dreamcast.hk^",
-        "||hdslb.com^"
+        "||hdslb.com^",
+        "||hdslb.org^",
+        "||im9.com^",
+        "||maoercdn.com^",
+        "||mincdn.com^",
+        "||yo9.com^",
+        "/^upos-hz-mirrorakam\\.akamaized\\.net$/"
```
Comment (ameshkov) [inline assets/services.json:268]: Replace with a regular rule please

File: assets/services.json [hunk 6]
```diff
@@ -114,12 +330,33 @@
...
+        "||discordpartygames.com^",
+        "||discordsays.com^",
+        "||discordstatus.com^",
+        "||airhorn.solutions^",
+        "||airhornbot.com^",
+        "||bigbeans.solutions^",
+        "||watchanimeattheoffice.com^",
+        "||discordapp.io^",
+        "||discordcdn.com^",
+        "/^hammerandchisel\\.ssl\\.zendesk\\.com$/"
```
Comment (ameshkov) [inline assets/services.json:359]: Please replace with a regular rule

File: assets/services.json [hunk 7]
```diff
@@ -151,31 +389,326 @@
...
+        "||xn--qoq462m.com^",
+        "||xn--tkry91n.com^",
+        "||xn--ubt498knmf.com^",
+        "||xn--xsq421m.com^",
+        "||xn--xsq605n.com^",
+        "||xn--xsq959n.com^",
+        "||xn--yf1at58a.com^",
+        "||xxbay.com^",
+        "||yibei.org^",
+        "/^ebay-25-assets\\.s3-us-west-1\\.amazonaws\\.com$/"
```
Comment (ameshkov) [inline assets/services.json:711]: Please replace with a regular rule

File: assets/services.json [hunk 8]
```diff
@@ -186,27 +719,461 @@
         "||epicgames.com^",
         "||easyanticheat.net^",
         "||easy.ac^",
-        "||eac-cdn.com^"
+        "||eac-cdn.com^",
+        "/^cdn\\d-epicgames-\\d+\\.file\\.myqcloud\\.com$/",
```
Comment (ameshkov) [inline assets/services.json:723]: Can all be replaced with regular rules

File: assets/services.json [hunk 9]
```diff
@@ -186,27 +719,461 @@
...
+        "||workplaceusecases.com^",
+        "||worldhack.com^",
+        "||www-facebook.com^",
+        "||wwwfacebok.com^",
+        "||wwwfacebook.com^",
+        "||wwwmfacebook.com^",
+        "||yogalayout.com^",
+        "||zuckerberg.com^",
+        "||zuckerberg.net^",
+        "/^fbcdn-a\\.akamaihd\\.net$/"
```
Comment (ameshkov) [inline assets/services.json:1175]: Can be replaced with a regular rule

**Review by ameshkov — CHANGES_REQUESTED** [review]
Thank you for the pull request! Could you please fix a few things before we merge it?

File: assets/services.json (hunk 1 above)
Comment (rootmelo92118) [reply assets/services.json]: I removed all domain names which are belonging to `imdb`.

Comment (rootmelo92118) [reply assets/services.json:157]: https://icannwiki.org/.imdb

File: assets/services.json (hunk 2 above)
Comment (rootmelo92118) [reply assets/services.json:207]: https://icannwiki.org/.zappos

File: assets/services.json (hunk 3 above)
Comment (rootmelo92118) [reply assets/services.json:208]: Of course.

File: assets/services.json (hunk 4 above)
Comment (rootmelo92118) [reply assets/services.json:217]: Ok, but I want to ask, why avoid using regex?

File: assets/services.json (hunk 5 above)
Comment (rootmelo92118) [reply assets/services.json:268]: Does Adblock-style syntax support not matching subdomains only matching the domain that I write?

File: assets/services.json (hunk 6 above)
Comment (rootmelo92118) [reply assets/services.json:355]: Finish

File: assets/services.json (hunk 5 above)
Comment (rootmelo92118) [reply assets/services.json:264]: Finish

File: assets/services.json (hunk 9 above)
Comment (rootmelo92118) [reply assets/services.json]: Finished

File: assets/services.json (hunk 8 above)
Comment (rootmelo92118) [reply assets/services.json]: Finished

File: assets/services.json (hunk 7 above)
Comment (rootmelo92118) [reply assets/services.json]: Finished

**Review by ameshkov — APPROVED** [review]

**Review by ameshkov — APPROVED** [review]

**Pull request merged**